.github/drafts/PULL_REQUEST_TEMPLATE/new_feature.md

```diff
@@ -7,11 +7,19 @@ assignees: ''
 
 ---
 
-## This PR adds...
+## Double check these details before you open a PR**
+*Tick the checkboxes to ensure you've done everything*
+- [] PR does not match another non-stale PR currently opened
+- [] PR name matches the format *New Feature: brief description of feature*
+
+
+## This PR adds/fixes...
 
 *List your features here and the benefits they bring.*
```

Comment:
```suggestion
<!-- List your features here and the benefits they bring. -->
```

Reply:
Why do we comment out this line though? Shouldn't the user see this?

Reply:
> Why do we comment out this line, though? Shouldn't the user see this?

This works like in other programming languages: comments cannot be read by computers, but they can be read by the user. The purpose of the comments should only be read by the PR creator, not by other users. Hope I get to understand. 🙂

Reply:
I think that line should be there as an instruction to people opening a PR into the repo. If it's only readable to us, people might get confused. 

Will this line shows up when someone open a PR into the repo? If not, I think we should stick with the old one

Reply:
Hmm, I think you misunderstood, sorry. 😅

When you use comments in Markdown, that's only readable when editing a comment or a Markdown file, but that can't be read when looking at the file/comment preview. Hope I get to understand now. 😅

Reply:
I see. In that case, I think we should stick with what we had. I want the people opening PRs to see that instruction and not have it hidden. Let me know if you had something else in mind for this. If not, I'll dismiss this in a few days

Reply:
Hmm, I think you misunderstood again... 😅
Try to use this code in a comment:

```markdown
<!-- List your features here and the benefits they bring. -->
```

Before submitting it, click on the `Preview` button, check the content, and then go back to the `Write` button.